### PR TITLE
Added support for SkiffOS

### LINKED ISSUE #7: Script does not show information on OS X
When I run the script, everything is properly formatted, but no information is printed for most items. 

Missing:
- OS information
- Uptime
- Packages
- Window Manager
- CPU
- Memory
- Song (I assume because of the mpc dependency)
- Image (also probably because of missing dependencies)

Color blocks print successfully.

Here's a picture:
http://imgur.com/2i9zujy

Terminal is iTerm2 (script runs the same on the default Terminal app) and I'm running OS X 10.11.2

### LINKED ISSUE #1657: Distro request: SkiffOS and Buildroot
## Description

Distro Request: SkiffOS and Buildroot

(SkiffOS is a distribution produced with the Buildroot tool).

Implemented in PR: #1664 

image

Homepage: https://github.com/skiffos/skiffos

Buildroot homepage: https://buildroot.org

Supports any target system using Buildroot to produce a consistent minimal OS for containers.

Support for neofetch [has been added](https://patchwork.ozlabs.org/project/buildroot/patch/20210115193539.471638-1-christian@paral.in/) -

BR2_PACKAGE_NEOFETCH=y

### PR DESCRIPTION
**Pull request by paralin**
https://github.com/skiffos/skiffos

Fixes #1657

### CODE REVIEW

**Comment by paralin** [comment]
image

**Comment by paralin** [comment]
image

**Comment by paralin** [comment]
image

**Comment by paralin** [comment]
image

**Comment by koutsie** [comment]
LGTM!

File: neofetch [hunk 1]
```diff
@@ -993,6 +993,12 @@ get_distro() {
                     *) distro="Siduction ($(lsb_release -sic))"
                 esac
 
+            elif [[ -f /etc/skiffos-release ]]; then
+                case $distro_shorthand in
+                    on|tiny) distro=SkiffOS ;;
+                    *) distro="SkiffOS ($(cat /etc/skiff-release | grep PRETTY_NAME | awk -F'"' '$0=$2'))"
+                esac
+
```
Comment (dylanaraps) [inline neofetch:1001]: Existing detection methods don't work?

Comment (paralin) [reply neofetch:1001]: /etc/os-release:

```
NAME=SkiffOS
VERSION=2021.02.2-31-g9badfd5
ID=skiff
VERSION_ID=9badfd5
PRETTY_NAME="SkiffOS 2021.02.2-31-g9badfd5"
BUILD_DATE="Thu Jun 10 17:11:44 PDT 2021"
BUILD_USER="paralin@forge"
VERSION_FULL="SkiffOS version 2021.02.2-31-g9badfd5 (paralin@forge) Thu Jun 10 17:11:44 PDT 2021"
```

/etc/skiff-release

```
NAME=SkiffOS
VERSION=2021.02.2-31-g9badfd5
ID=skiff
VERSION_ID=9badfd5
PRETTY_NAME="SkiffOS 2021.02.2-31-g9badfd5"
BUILD_DATE="Thu Jun 10 17:11:44 PDT 2021"
BUILD_USER="paralin@forge"
VERSION_FULL="SkiffOS version 2021.02.2-31-g9badfd5 (paralin@forge) Thu Jun 10 17:11:44 PDT 2021"
BUILD_CONFIG="pine64/common,skiff/moduleimg,pine64/phone,apps/docker,skiff/core,core/gentoo"
```

Comment (paralin) [reply neofetch:1001]: The current `master` fails on my end, I bisected it to commit `41f75b7b484cca653e0db19b194d99a993dbf5f8`

neofetch: line 6364: syntax error near unexpected token `)'
neofetch: line 6364: `        "CelOS"*)'

Comment (paralin) [reply neofetch:1001]: https://github.com/dylanaraps/neofetch/pull/1772 fixed here

Comment (paralin) [reply neofetch]: OK @dylanaraps I simplified it to use existing os-release detection, works fine.

**Comment by dylanaraps** [comment]
Thanks again.

**Pull request merged**